app/forms/support_interface/multiple_provider_users_wizard.rb

```diff
@@ -0,0 +1,161 @@
+module SupportInterface
+  class MultipleProviderUsersWizard
+    include ActiveModel::Model
+    include ActiveModel::Validations
+
+    attr_accessor :provider_id, :state_store, :provider_users, :stored_provider_users, :position
+    validates :provider_users, presence: true
+
+    def self.build(state_store:, provider_id:, position: nil)
+      state = state_store.read
+      stored_provider_users = JSON.parse(state)['provider_users'] if state
+      create_csv_row = ->(pu) { [pu['first_name'], pu['last_name'], pu['email_address']].compact.join(',') + "\n" }
```

Comment:
could we move this to a dedicated method?

Reply:
Past the first step of saving the state of the users inputed, do we need to keep the reading/writing the format in csv?

Reply:
Good point - will fix

Reply:
Fix - https://github.com/DFE-Digital/apply-for-teacher-training/pull/4896/commits/2ba1b1bbf5fd0ef4fc6c7568ad343144ab12fe41